.github/workflows/nightwatch-testing-library.yml

```
@@ -12,28 +12,28 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [12, 10]
+        node: [10, 12, 14, 16]
```

10 is also EOL now, so we can remove it and merge
```suggestion
        node: [12, 14, 16]
```

---

We should also change `package.json` then, no?

---

We don't seem to have any `engines` configuration. We could add it, but I think that should be in a major release, since it's a breaking change for anyone using a package manager with strict peer dependency checks.